### DISCUSSION TITLE
Is there a way to take censoring option to below 1?

### DISCUSSION THREAD

**Discussion by Nailerisnt** [silveredgold/beta-protection#41] [Q&A] [answered]
I would like to go to take the "pixels" setting to less than 1 for some areas like the face. Is there anyway to input decimal values below 1 on the settings scale. You can input decimals above 1 when importing settings but the bottom of the input scale seems to be 1. 

Any ideas or could this be added in the future?

**Comment by silveredgold** [comment]
Not currently planned. I'll look into whether this might work in the future, but there are a few other components behind the scenes that assume the censoring level will always be whole numbers.

**Comment by silveredgold** [comment, marked as answer]
Looping back to this, I won't be able to implement this. So many individual parts of the system assume whole numbers that we won't be able to do decimals. If you're just wanting to drop the strength for a certain body part (and you're using the Beta Censoring backend), you can use the `ClassStrength` option in the configuration file to do this ([as explained in the docs here](https://silveredgold.github.io/beta-censoring/content/configuration.html#other-configuration))
